docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -810,6 +810,10 @@ children:
     url: Understanding-the-differences.html
     output: 'web, pdf'
 
+  - title: 'Booting process'
+    url: LB3-vs-LB4-booting.html
```

Comment:
Since this new content is related to migration from LB3, please move it to `docs/site/migration` directory.

I see we already have `docs/site/migration/boot-scripts.md`, what is the relation between that document and this new page? Should you perhaps add a link from `6. Running boot scripts` to that migration guide?

I think that's a general rule you should follow - for each section (LB4 boot step), add a link to the relevant migration guide.

Reply:
> I see we already have docs/site/migration/boot-scripts.md, what is the relation between that document and this new page?

The new page describes everything related to booting, `docs/site/migration/boot-scripts.md` is just about boot scripts. Added link.